README.md

```diff
@@ -27,165 +27,138 @@ and signs AMP documents as requested by the AMP Cache.
-or feel free to reach out in private if needed.
+  1. Run Chrome M70 or later (as of 2018-09-18, this is
+     [Beta](https://www.google.com/chrome/beta/) or
+     [Dev](https://www.google.com/chrome/dev/)). On the
+     command-line, pass the following flags:
+     ```
+     --user-data-dir=/tmp/udd
+     --ignore-certificate-errors-spki-list=$(openssl x509 -pubkey -noout -in path/to/fullchain.pem | openssl pkey -pubin -outform der | openssl dgst -sha256 -binary | base64)
+     --enable-features=SignedHTTPExchange
+     'data:text/html,<a href="https://localhost:8080/priv/doc/https://example.com/">click me'
+     ```
+  2. Open DevTools. Check 'Preserve log'.
+  3. Click the `click me` link.
+  4. Watch the URL transmogrify! Verify it came from an SXG by switching
+     DevTools to the Network tab and looking in the Size column for "(from
+     signed-exchange)".
+
+#### Demonstrate privacy-preserving prefetch
+
+This step is optional; just to show how [privacy-preserving
+prefetch](https://wicg.github.io/webpackage/draft-yasskin-webpackage-use-cases.html#private-prefetch)
+works with SXGs.
+
+  1. `go get -u github.com/ampproject/amppackager/cmd/amppkg_dl_sxg`.
+  2. `amppkg_dl_sxg https://localhost:8080/priv/doc/https://example.com/`
+  3. Stop `amppkg` with Ctrl-C.
+  4. `go get -u github.com/ampproject/amppackager/cmd/amppkg_test_cache`.
+  5. `amppkg_test_cache`
+  6. Open Chrome and DevTools.
+  7. Visit `https://localhost:8000/`. Observe the prefetch of `/test.sxg`.
+  8. Click the link. Observe that the cached SXG is used.
+
+#### Productionizing
+
+For now, productionizing is a bit manual. We would love to see scripts or
+packages that build in the best practices!
+
+The minimum steps are:
+
+  1. Don't pass `-development` flag to `amppkg`. This causes it to serve HTTP
+     rather than HTTPS.
+  2. Don't expose `amppkg` to the outside world; keep it on your internal
+     network.
+  3. Configure your TLS-serving frontend server to conditionally proxy to
+     `amppkg`, if any of:
+     1. The URL starts with `/amppkg/`.
+     2. The URL is AMP and the `AMP-Cache-Transform` request header is present.
+  4. Configure your frontend server to add `Vary: AMP-Cache-Transform` if the
+     URL is AMP. This should occur for both HTML and SXG responses.
+  5. Get an SXG cert from your CA. It must use an EC key with the prime256v1
+     algorithm, and it must have a [CanSignHttpExchanges
+     extension](https://wicg.github.io/webpackage/draft-yasskin-httpbis-origin-signed-exchanges-impl.html#cross-origin-cert-req).
+     You MUST use this in `amppkg.toml`, and MUST NOT use it in your frontend.
+
+You may also want to:
+
+  1. Launch `amppkg` as a restricted user.
+  2. Save its stdout to a rotated log somewhere.
+
+Once you've done the above, you should be able to launch Chrome without any
+comamndline flags; just make sure chrome://flags/#enable-signed-http-exchange is
+enabled.
+
+#### Replicating
```

Comment:
Redundancy?  I always think of replication in the context of data replication...